test/Prototypes/DoubleWidth.swift.gyb

```diff
@@ -464,6 +463,12 @@ extension DoubleWidth : FixedWidthInteger {
     return (quotient_, remainder_)
   }
 
+  public func isMultiple(of other: DoubleWidth) -> Bool {
```

Comment:
We can't piggyback on the default implementation here because `DoubleWidth` doesn't unconditionally conform to `SignedInteger` or `UnsignedInteger` -- it's one or the other, but we don't know which.